index.html

```diff
@@ -7,48 +7,167 @@
+              <li class="lang-list white">HTML</li>
+              <li class="lang-list white">CSS</li>
+            </ul>
+            <button class="button large" type="button">
+              <a href="#" target="_blank">See project</a>
+            </button>
+          </div>
+          <div class="placeholder1">
+            <h2 class="title1">Profesional Art Printing Data</h2>
+            <p class="description1">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="lang">
+              <li class="lang-list white">JavaScript</li>
+              <li class="lang-list white">HTML</li>
+              <li class="lang-list white">CSS</li>
+            </ul>
+            <button class="button large" type="button">
+              <a href="#" target="_blank">See project</a>
+            </button>
+          </div>
+          <div class="placeholder1">
+            <h2 class="title1">Profesional Art Printing Data</h2>
+            <p class="description1">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="lang">
+              <li class="lang-list white">JavaScript</li>
+              <li class="lang-list white">HTML</li>
+              <li class="lang-list white">CSS</li>
+            </ul>
+            <button class="button large" type="button">
+              <a href="#" target="_blank">See project</a>
+            </button>
+          </div>
+          <div class="placeholder1">
+            <h2 class="title1">Profesional Art Printing Data</h2>
+            <p class="description1">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="lang">
+              <li class="lang-list white">JavaScript</li>
+              <li class="lang-list white">HTML</li>
+              <li class="lang-list white">CSS</li>
+            </ul>
+            <button class="button large" type="button">
+              <a href="#" target="_blank">See project</a>
+            </button>
+          </div>
+          <div class="placeholder1">
+            <h2 class="title1">Profesional Art Printing Data</h2>
+            <p class="description1">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="lang">
+              <li class="lang-list white">JavaScript</li>
+              <li class="lang-list white">HTML</li>
+              <li class="lang-list white">CSS</li>
+            </ul>
+            <button class="button large" type="button">
+              <a href="#" target="_blank">See project</a>
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="container">
+        <h1 class="aboutme">About me</h1>
```

Comment:
Kindly, I suggest you use a `h2` tag instead of the `h1`, because `h1` can be used only for First Heading. 😄